pkg/cmd/roachtest/cluster.go

```diff
@@ -2846,6 +2852,37 @@ func (c *clusterImpl) StopGrafana(ctx context.Context, l *logger.Logger, dumpDir
 	return roachprod.StopGrafana(ctx, l, c.name, dumpDir)
 }
 
+// AddGrafanaAnnotation creates a grafana annotation. If internal is false, it creates
+// an annotation for the centralized grafana instance. If true, it creates the annotation
+// for the internal grafana instance spun up in roachtests through StartGrafana.
+func (c *clusterImpl) AddGrafanaAnnotation(
+	ctx context.Context, l *logger.Logger, internal bool, req grafana.AddAnnotationRequest,
```

Comment:
I wonder if we should simplify the signature since the most common use-case is against the centralized instance. Since Go doesn't support function overloading, we could drop `internal`, and create another `func`, say `AddInternalGrafanaAnnotation` for the uncommon case.

Reply:
Sure, I like that idea. Done.